libs/src/utils/utils.ts

```diff
@@ -226,5 +226,11 @@ export class Utils {
     return await Promise.race([promise, timeoutPromise])
   }
 
+  static getRandomInt(min: number, max: number): number {
```

Comment:
we can dedupe the random function in entityManager.ts

Reply:
Yeah I figured we can do later bc this is now used in the client